app/src/main/res/layout/activity_main.xml

```diff
@@ -6,6 +6,13 @@
     android:layout_height="match_parent"
     tools:viewBindingIgnore="true">
 
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:adjustViewBounds="true"
+        android:scaleType="fitCenter"
+        android:src="@drawable/ic_launcher_foreground" />
```

Comment:
It's a bit huge imo, would be better to give it a specific size.

Reply:
> It's a bit huge imo, would be better to give it a specific size.

For example?

### Original jellyfin-web splash screen
![qemu-system-x86_64_HA2v5pOwUD](https://user-images.githubusercontent.com/25919231/122109947-90783780-cde3-11eb-83af-6112e0350498.png)
![qemu-system-x86_64_mJvGNuxwLV](https://user-images.githubusercontent.com/25919231/122109950-91a96480-cde3-11eb-9ef0-ae7bf8a76fe9.png)

### Current PR
![qemu-system-x86_64_frIjxPnNdh](https://user-images.githubusercontent.com/25919231/122110002-a4239e00-cde3-11eb-80f1-9d79783e0086.png)
![qemu-system-x86_64_5AP8l2aNyi](https://user-images.githubusercontent.com/25919231/122109999-a38b0780-cde3-11eb-908c-8f4981d7566e.png)

### Current PR + Padding 64dp
![qemu-system-x86_64_fHJc9BukHT](https://user-images.githubusercontent.com/25919231/122110085-c1586c80-cde3-11eb-86fa-17cf003b5699.png)
![qemu-system-x86_64_3ics5wUvtV](https://user-images.githubusercontent.com/25919231/122110082-c0bfd600-cde3-11eb-847c-7e615a60f97b.png)
![studio64_rxE2VbQdfh](https://user-images.githubusercontent.com/25919231/122111988-f796eb80-cde5-11eb-8e1a-ec788cca9edd.png)


### Current PR + width 256dp
![studio64_HUpXzxzZ2U](https://user-images.githubusercontent.com/25919231/122112023-01205380-cde6-11eb-92dc-880791a7770b.png)
![studio64_ExoVJRVHDq](https://user-images.githubusercontent.com/25919231/122112036-04b3da80-cde6-11eb-8300-1d5b352e306c.png)


Or anything else?

Reply:
The problem is that the current launcher icon includes quite a bit of padding itself. Good question, I'll probably have to play with the layout myself on-device.